packages/compass-collection/src/stores/collection-tab.spec.ts

```diff
@@ -18,6 +18,7 @@ const defaultMetadata = {
 
 const defaultTabOptions = {
   namespace: defaultMetadata.namespace,
+  subTab: 'Documents',
```

Comment:
This should be `as const` to avoid doing unneccessary as any cast below (it's always a good practice to limit those to a minimum and actually resolve TS errors if possible)

```suggestion
  subTab: 'Documents' as const,
```